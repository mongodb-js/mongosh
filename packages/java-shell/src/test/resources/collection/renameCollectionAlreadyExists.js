// before
db.coll.drop();
db.newName.drop();
db.coll.insertOne({a: 1});
db.newName.insertOne({a: 2});
// command
db.coll.renameCollection('newName');
// command
db.coll.find();
// command
db.newName.find();
// clear
db.coll.drop();
db.newName.drop();
