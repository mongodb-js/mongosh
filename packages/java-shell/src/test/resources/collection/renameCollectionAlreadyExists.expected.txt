com.mongodb.MongoCommandException: Command failed with error 48 (NamespaceExists): 'target namespace exists' on server %mongohostport%. The full response is {"ok": 0.0, "errmsg": "target namespace exists", "code": 48, "codeName": "NamespaceExists"}
[ { "_id": <ObjectID>, "a": 1 } ]
[ { "_id": <ObjectID>, "a": 2 } ]